test/controllers/controllershelpers/HomeCardGeneratorSpec.scala

```diff
@@ -187,25 +207,63 @@ class HomeCardGeneratorSpec extends ViewSpec with MockitoSugar {
   }
 
   "Calling getNationalInsuranceCard" must {
-    "return NI Card when toggled on" in {
+    "return NI Card when toggled on and NISP toggle is disabled" in {
       when(mockFeatureFlagService.get(ArgumentMatchers.eq(NationalInsuranceTileToggle)))
         .thenReturn(Future.successful(FeatureFlag(NationalInsuranceTileToggle, isEnabled = true)))
 
+      when(mockFeatureFlagService.get(ArgumentMatchers.eq(NiAndSpMergeTileToggle)))
+        .thenReturn(Future.successful(FeatureFlag(NiAndSpMergeTileToggle, isEnabled = false)))
+
       lazy val cardBody = homeCardGenerator.getNationalInsuranceCard().futureValue
 
       cardBody mustBe Some(nationalInsurance())
     }
 
-    "return None when toggled off" in {
+    "return None when NationalInsuranceTileToggle is disabled and NiAndSpMergeTileToggle is disabled" in {
```

Comment:
We'd need a test case for when the nisp merge tile toggle is enabled that the NI Toggle returns None